blog/2020-09-07-OptaPlannerWeek2020.adoc

```diff
@@ -0,0 +1,78 @@
+= OptaPlanner Week 2020 recordings
+:page-interpolate: true
+:awestruct-author: ge0ffrey
+:awestruct-layout: blogPostBase
+:awestruct-tags: [feature, coding]
+:awestruct-share_image_filename: constraintStreamGroupBy.png
+
+It's a wrap! Last week, during the online *OptaPlanner Week* event,
+we had 10 talks by 13 speakers across 3 days with - per day -
+up to 250+ live attendees and 1300+ viewers afterwards.
+We've split up the talks into separate videos for your convenience, watch them below.
+
+Big thanks to all speakers for their superb content,
+all attendees for their great questions during the sessions
+and especially Karina Varela for organizing, hosting and moderating this event perfectly.
+
+image::optaPlannerWeek2020Speakers.jpg[]
+
+The live questions really improved the interactively with the audience.
```

Comment:
Not sure what you're trying to say here. Perhaps `interaction`? `engagement of the audience`?